dataflow/build.gradle

```diff
@@ -64,7 +64,7 @@ createDataflowShaded('errorprone')
 def testDataflowAnalysis(taskName, dirName, className) {
```

Comment:
Regarding @wmdietl's comment, I think it's not just about the test gradle file, but also the java code that executes the test. Is it possible to define a util class for `ReachDefinitions` and other test entry points, so that in their main function they can simply call `runTest` from the util class.

Reply:
Thanks!

Reply:
Yes, no changes to this file and the live-var tests should be done. Just extract the common logic from https://github.com/eisop/checker-framework/blob/master/dataflow/src/test/java/livevar/LiveVariable.java (and possibly https://github.com/eisop/checker-framework/blob/master/dataflow/src/main/java/org/checkerframework/dataflow/cfg/playground/LiveVariablePlayground.java) and make it easier to add a new DFA test.